.circleci/config.yml

```diff
@@ -15,6 +15,9 @@ parameters:
   run-langsmith-evals:
     type: boolean
     default: false
+  run-custom-command-evals:
```

Comment:
This is just a suggestion based on preference, I'd probably always use `run-custom-evals` instead of `run-custom-command-evals` everywhere